ibmcloud/terraform/cluster/variables.tf

```diff
@@ -64,4 +64,9 @@ variable "ansible_dir" {
 variable "scripts_dir" {
     description = "Subdirectory for shell scripts"
     default = "./scripts"
+}
+
+variable "cri_runtime_endpoint" {
+    description = "cri-runtime-endpoint for Ansible playbook, inventory and vars files"
+    default = "/run/containerd/containerd.sock"
```

Comment:
The variable `cri_runtime_endpoint` is defined in `start-cloud-api-adaptor/variables.tf`.

Is this definition necessary also in `cluster/variables.tf`?

Reply:
@yoheiueda I'm not experts for terraform, but I noticed the variables are all duplicated in https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/ibmcloud/terraform/cluster/variables.tf and https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/ibmcloud/terraform/start-cloud-api-adaptor/variables.tf, is that because `cluster` or `start-cloud-api-adaptor` can be run separately?

Reply:
I think variables that to be used in a directory should be defined in the same directory.

Regarding the `cluster` directory, the variable `cri_runtime_endpoint` is not used in the same directory.

Reply:
Read the terraform components carefully, yes, I think we don't need it in `cluster`. Revise a little in `start-cloud-api-adaptor` again. @yoheiueda